test/parser-recovery.js

```diff
@@ -308,87 +308,87 @@ describe('Invoke parser with incomplete expression', function() {
+                                            "type": "operator",
+                                            "value": ".",
+                                            "position": 42
+                                        },
+                                        {
+                                            "type": "name",
+                                            "value": "Text",
+                                            "position": 46
+                                        }
+                                    ],
+                                    "type": "error",
+                                    "predicate": [
+                                        {
+                                            "type": "filter",
+                                            "expr": {
+                                                "type": "error",
+                                                "error": {
+                                                    "code": "S0207",
+                                                    "position": 46,
+                                                    "token": "(end)"
+                                                }
                                             },
-                                            {
-                                                "type": "name",
-                                                "value": "Text",
-                                                "position": 46
-                                            }
-                                        ],
-                                        "type": "error",
-                                        "predicate": [
-                                            {
-                                                "type": "filter",
-                                                "expr": {
-                                                    "type": "error",
-                                                    "error": {
-                                                        "code": "S0207",
-                                                        "position": 46,
-                                                        "token": "(end)"
-                                                    }
-                                                },
-                                                "position": 39
-                                            }
-                                        ]
-                                    }
+                                            "position": 39
+                                        }
+                                    ]
                                 }
-                            ],
-                            "position": 36
-                        }
-                    ]
-                };
+                            }
+                        ],
+                        "position": 36
+                    }
+                ]
+            };
```

Comment:
should we push this object 4 spaces to the right to match the original?

Reply:
Trying to make that happen, not sure why my editor reverts those changes